.devcontainer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM library/node:lts-bookworm
+
+ARG DEBIAN_FRONTEND=noninteractive
+RUN apt update && \
+    apt-get install -y nodejs nasm gdb unzip p7zip-full openjdk-17-jre wget python3 qemu-system-x86 git-core build-essential libc6-dev-i386-cross libc6-dev-i386 clang curl time
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
+RUN export PATH="$HOME/.cargo/bin:$PATH" && \
+    rustup toolchain install stable && \
+    rustup target add wasm32-unknown-unknown && \
+    rustup component add rustfmt-preview
+RUN wget -P images/ https://k.copy.sh/{linux.iso,linux4.iso,buildroot-bzimage.bin,openbsd-floppy.img,kolibri.img,windows101.img,os8.img,freedos722.img}
```

Comment:
I don't think you can use `{}` in Dockerfiles:

```
 > [dev_container_auto_added_stage_label 5/7] RUN wget -P images/ https://k.********.sh/{linux.iso,linux4.iso,buildroot-bzimage.bin,openbsd-floppy.img,kolibri.img,windows101.img,os8.img,freedos722.img}:
0.473 --2024-01-02 22:20:11--  https://k.********.sh/%7Blinux.iso,linux4.iso,buildroot-bzimage.bin,openbsd-floppy.img,kolibri.img,windows101.img,os8.img,freedos722.img%7D
404 Not Found
0.887 2024-01-02 22:20:11 ERROR 404: Not Found.
0.887 
```

Also, I don't believe these files will end up in the correct `images/` directory. The `images/` directory is inside of the repo, which isn't available when the container is built.

Reply:
This is done as well. I moved it to a task instead of building during Dockerfile startup. I also tested fresh installs with both a Docker remote container and a Github codespace.